### PR TITLE
DT-1832: Remove use of EPUI

### PR DESCRIPTION
**Pull request by nt-gt**

### CODE REVIEW

File: ebl/src/main/java/org/dcsa/conformance/standards/ebl/checks/EblDatasets.java [hunk 1]
```diff
@@ -63,4 +53,28 @@ public class EblDatasets {
...
+    "TRAC",
+    "BRIT",
+    "GSBN",
+    "WISE",
+    "GLEIF",
+    "W3C",
+    "DNB",
+    "FMC",
+    "DCSA",
+    "EU",
```
Comment (palatsangeetha) [inline ebl/src/main/java/org/dcsa/conformance/standards/ebl/checks/EblDatasets.java:76]: As per the new snapshot the EU option has been removed from this list.

Comment (nt-gt) [inline ebl/src/main/java/org/dcsa/conformance/standards/ebl/checks/EblDatasets.java:76]:
```suggestion
```

Comment (nt-gt) [reply ebl/src/main/java/org/dcsa/conformance/standards/ebl/checks/EblDatasets.java]: Thanks, should be removed now. :)

**Review by palatsangeetha — APPROVED** [review]
The changes looks good to me.

**Review by preetamnpr — APPROVED** [review]

**Pull request merged**